dagger/scripts/ci_contract_validation.py

```
@@ -0,0 +1,633 @@
+        "  exit 0\n"
+        "fi\n"
+        f"exec {real_uname} \"$@\"\n"
+    )
+    uname_path.chmod(0o755)
+
+    env = os.environ.copy()
+    env["HOME"] = tmp
+    env["PATH"] = path_with_shims(shadow_path, tmp_path)
+
+    def run(*command):
+        return subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)